src/components/DuosHeader.js

```diff
@@ -118,15 +118,17 @@ export const headerTabsConfig = [
     link: '/member_console',
     search: 'member_console',
     children: [
-      { label: 'DAR Requests', link: '/member_console' }
+      { label: 'DAR Requests', link: '/member_console' },
+      { label: 'Datasets', link: '/dataset_catalog' },
     ],
     isRendered: (user) => user.isMember
   },
   {
     label: 'Researcher Console',
-    link: '/datalibrary',
-    search: 'datalibrary',
+    link: '/dataset_catalog',
+    search: 'dataset_catalog',
```

Comment:
Is the expectation that we set these back to the default?

Reply:
Not sure, I had just restored everything from before the removal— I'll check with Jonathan.

Reply:
Confirmed with Jonathan, setting the default back to the Data Catalog is the expectation!